src/main/webapp/app/exercises/programming/manage/grading/testcase-analysis/feedback-analysis.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { FeedbackAnalysisService, FeedbackDetailsWithResultIdsDTO, SimplifiedTask } from 'app/exercises/programming/manage/grading/testcase-analysis/feedback-analysis.service';
+import { Observable } from 'rxjs';
+import { concatMap, tap } from 'rxjs/operators';
+import { HttpResponse } from '@angular/common/http';
+import { AlertService } from 'app/core/util/alert.service';
+
+export interface FeedbackDetail {
+    count: number;
+    relativeCount: number;
+    detailText: string;
+    testCaseName: string;
+    task: number;
+}
+
+@Component({
+    selector: 'jhi-feedback-analysis',
+    templateUrl: './feedback-analysis.component.html',
+    standalone: true,
+    imports: [ArtemisSharedModule],
+    providers: [FeedbackAnalysisService],
+})
+export class FeedbackAnalysisComponent implements OnInit {
+    @Input() readonly exerciseTitle?: string;
+    @Input() readonly exerciseId?: number;
+    @Input() readonly isAtLeastEditor!: undefined | boolean;
+    resultIds: number[] = [];
+    tasks: SimplifiedTask[] = [];
+    feedbackDetails: FeedbackDetail[] = [];
+
+    constructor(
+        private simplifiedProgrammingExerciseTaskService: FeedbackAnalysisService,
+        private alertService: AlertService,
+    ) {}
+
+    ngOnInit(): void {
+        if (this.isAtLeastEditor) {
```

Comment:
isAtLeastEditor check added

Reply:
I don't think this is done anywhere else for the Artemis client in that position. Is it really sensible to add the additional authorization check in that position? Isn't that what the routes are used for? That way we used to only have to update the authorization in one place if we wanted to change something.